Joker/src/components/tab-bar.tsx

```diff
@@ -0,0 +1,55 @@
+// import {RouteProp} from '@react-navigation/native';
+import {BottomTabNavigationEventMap} from '@react-navigation/bottom-tabs';
+import {
+  NavigationHelpers,
+  ParamListBase,
+  TabNavigationState,
+} from '@react-navigation/native';
+import React, {useCallback} from 'react';
+import {StyleSheet, Text, TouchableOpacity, View} from 'react-native';
+import {useSafeAreaInsets} from 'react-native-safe-area-context';
+import {IS_IOS} from '../variables';
+
+type TabbarProps = {
+  navigation: NavigationHelpers<ParamListBase, BottomTabNavigationEventMap>;
```

Comment:
What kind of NavigationHelpers??
Use typed navigation hook

Reply:
done